### PR TITLE
goxel: init at 0.7.2

### PR DESCRIPTION
**Pull request by tilpner**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by tilpner** [comment]
This is advertised to run on macOS, but I can't test it there. I'll remove `++ darwin` from `meta.platforms` if I can't find someone to test it there.

This shouldn't be merged until it's either been tested on darwin or darwin has been removed from platforms.

**Comment by tilpner** [comment]
I found someone to build this on darwin, but it doesn't find argp.h, even after adding gnulib to buildInputs. It's probably better for someone on darwin to fix this, I've removed the platform for now.

Ready to merge now! :)

File: pkgs/applications/graphics/goxel/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchFromGitHub, scons, pkgconfig, wrapGAppsHook
+, glfw3, gtk3, libpng12 }:
+
+stdenv.mkDerivation rec {
+  name = "goxel-${version}";
+  version = "v0.7.2";
```
Comment (nlewo) [inline pkgs/applications/graphics/goxel/default.nix]: Version in the package name must start with a digit.

**Review by nlewo — CHANGES_REQUESTED** [review]
Execution fails for me:
```
./result/bin/goxel
./result/bin/goxel: relocation error: /nix/store/q3wx1gab2ysnk5nyvyyg56ana2v4r2ar-glibc-2.24/lib/libpthread.so.0: symbol __libc_dl_error_tsd, version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference
```

**Comment by tilpner** [comment]
@nlewo I don't know how to reproduce or fix that. Did you do something like in #23203? Does something change when you add glibc to buildInputs?

**Comment by adisbladis** [comment]
@GrahamcOfBorg build goxel

**Pull request merged**